src/irmin-pack/store_intf.ml

```diff
@@ -91,7 +91,4 @@ module type Sigs = sig
   module type Specific = Specific
   module type S = S
   module type Maker = Maker
-
```

Comment:
`store.ml` is now empty. I suppose you are about to delete the file.

Reply:
Indeed, see #1379  :-)